packages/fetch-api/src/request.ts

```diff
@@ -10,9 +10,10 @@ const requestMap: { [key: string]: Promise<Result> } = {}
 function request(
     url: string,
     normalizer: (a: any, pingbackType?: PingbackEventType) => any = identity,
-    pingbackType?: PingbackEventType
+    pingbackType?: PingbackEventType,
+    noCache: boolean = false
 ) {
-    if (!requestMap[url]) {
+    if (!requestMap[url] || noCache) {
```

Comment:
Interesting - what effect was this having?

Reply:
@nicksantan we cache the requests in memory in a map based on url, the random endpoint always has the same url and needs the `noCache` param to skip the check `!requestMap[url]`